dockers/docker-fpm-quagga/start.sh

```diff
@@ -21,6 +18,8 @@ echo "# Config files managed by sonic-config-engine" > /var/sonic/config_status
 
 rm -f /var/run/rsyslogd.pid
 
+supervisorctl start bgpcfgd
```

Comment:
Will resolve in later PRs.

---
In reply to: [130536078](https://github.com/Azure/sonic-buildimage/pull/808#discussion_r130536078) [](ancestors = 130536078)